specs/mgt-people-picker.md

```diff
@@ -46,6 +46,114 @@ The people picker components provides a way for developers to select users, grou
 <mgt-people-picker group-ids="02bd9fd6-8f93-4758-87c3-1fb73740a315,06f62f70-9827-4e6e-93ef-8e0f2d9b7b23"></mgt-people-picker>
 ```
 
+## Form validation
+
+
+The mgt-people-picker component can be used as a form control in a form, and the form can be submitted with the attributes as part of the form data. It makes use of the form-associated custom elements API, which provides a new set of capabilities that make custom controls work like built-in form controls. The component will implement the form-associated behaviors to participate in form submission, validation, and other form-related behaviors. [Read more](https://docs.google.com/document/d/1JO8puctCSpW-ZYGU8lF-h4FWRIDQNDVexzHoOQ2iQmY/edit?pli=1#heading=h.2hgix04sc53t) about the form-associated custom elements API and [how to create custom form controls](https://css-tricks.com/creating-custom-form-controls-with-elementinternals/).
+
+The component can be used with the following attributes, which can then be validated as part of the form submission. 
+
+
+| Attribute          | Description                                                                                                 | Implementation                                                                                                      |
```

Comment:
let's use the properties on the object instead as the attributes are inputs while the selectedPeople property changes value as the user interacts with the control.